common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -201,7 +208,7 @@
                     it('when clicking on link with current language', function () {
                         state = jasmine.initializePlayer();
                         var Caption = state.videoCaption,
-                            link = $('.langs-list li[data-lang-code="en"] a');
+                            link = $('.langs-list li[data-lang-code="en"] .control-lang');
```

Comment:
Are both `.control-lang` and `.control` correct in this case?  The test above uses `.control`.